piper/ratbagd.py

```diff
@@ -59,6 +59,46 @@ class RatbagdDBusUnavailable(BaseException):
     pass
 
 
+class RatbagdDBusTimeout(BaseException):
+    """Signals that a timeout occurred during a DBus method call."""
+    pass
+
+
+class RatbagErrorDevice(BaseException):
+    """An exception corresponding to RatbagErrorCode.RATBAG_ERROR_DEVICE."""
+    pass
+
+
+class RatbagErrorCapability(BaseException):
+    """An exception corresponding to RatbagErrorCode.RATBAG_ERROR_CAPABILITY."""
+    pass
+
+
+class RatbagErrorValue(BaseException):
+    """An exception corresponding to RatbagErrorCode.RATBAG_ERROR_Value."""
+    pass
+
+
+class RatbagErrorSystem(BaseException):
+    """An exception corresponding to RatbagErrorCode.RATBAG_ERROR_System."""
+    pass
+
+
+class RatbagErrorImplementation(BaseException):
+    """An exception corresponding to RatbagErrorCode.RATBAG_ERROR_IMPLEMENTATION."""
+    pass
```

Comment:
might want to make a general RatbagError and then derive from that one, just so callers can have a `except RatbagError` clause to catch all the specific ones. But we can add that later.